README.md

```diff
@@ -16,3 +16,15 @@ A weather forecasting app built on node.js (express.js).
    ```bash
    npm run test
    ```
+
+## Screen Shots
+### Weather page
```

Comment:
You don't need to add screenshots of all the pages. You can simply add two "full-window screenshot" of the homepage (with and without search results). You can also use a single full-page GIF here, demonstrating how the app works. With those, you can get rid of the other screenshots accordingly.

Reply:
Gif added showing how the app works